### PR TITLE
Get all visible markers

### PR DESCRIPTION
**Pull request by jeffb-sfdc**
renamed getAllMarkers() to getAllVisibleMarkers(), and added getCountBadge()

### CODE REVIEW

File: locators/lib/1.37.0.ts
Comment (jeffb-sfdc) [inline locators/lib/1.37.0.ts:55]: I added a new function (`getCountBadge()`) and needed an addition locator

File: page-objects/src/components/bottomBar/ProblemsView.ts
Comment (jeffb-sfdc) [inline page-objects/src/components/bottomBar/ProblemsView.ts:16]: I run with the Code Spell Checker extension installed, and it found this typo

File: page-objects/src/components/bottomBar/ProblemsView.ts
Comment (jeffb-sfdc) [inline page-objects/src/components/bottomBar/ProblemsView.ts:49]: Added `getAllMarkers()`

File: page-objects/src/components/bottomBar/ProblemsView.ts
Comment (jeffb-sfdc) [inline page-objects/src/components/bottomBar/ProblemsView.ts:62]: And renamed `getAllMarkers()` to `getAllVisibleMarkers()`

File: page-objects/src/components/bottomBar/ProblemsView.ts
Comment (jeffb-sfdc) [inline page-objects/src/components/bottomBar/ProblemsView.ts:79]: Added `getCountBadge()`

File: sample-projects/helloworld-sample/src/ui-test/bottomBar-test.ts
Comment (jeffb-sfdc) [inline sample-projects/helloworld-sample/src/ui-test/bottomBar-test.ts:47]: I commented out these tests, as they aren't reliable.

File: sample-projects/helloworld-sample/src/ui-test/bottomBar-test.ts
Comment (jeffb-sfdc) [inline sample-projects/helloworld-sample/src/ui-test/bottomBar-test.ts:76]: And in their place I added this test.

File: page-objects/src/components/bottomBar/ProblemsView.ts
Comment (mlorinc) [reply page-objects/src/components/bottomBar/ProblemsView.ts:79]: Could you please add another function which would return Promise<number>.

File: sample-projects/helloworld-sample/src/ui-test/bottomBar-test.ts
Comment (mlorinc) [reply sample-projects/helloworld-sample/src/ui-test/bottomBar-test.ts:47]: I think we should use it.skip(...) instead of commenting out tests. Mocha will show tests in the output but tests will not be executed. Reason comments are good and I believe we should keep them.

**Review by mlorinc — COMMENTED** [review]
Thank you for your PR! I agree with every change you have done except commenting out tests. In this area I am open for another opinions from other participants.

File: sample-projects/helloworld-sample/src/ui-test/bottomBar-test.ts
Comment (odockal) [reply sample-projects/helloworld-sample/src/ui-test/bottomBar-test.ts:47]: Ok. I agree in using it.skip (which is good idea Marian, btw.), at the same time, we should come up with an issue, that will tackle skipped tests in such test environment (hard coded resolution, exact number of markers that are actually visible) allowing us to cover all the provided implementation.

**Review by odockal — APPROVED** [review]

**Comment by odockal** [comment]
@jeffb-sfdc thanks for PR, it is nice. I agree with Marian, we should come up with issue that will eventually tackle skipped tests in controlled test env. so that tests omitted now are back and reliable.

File: page-objects/src/components/bottomBar/ProblemsView.ts
Comment (jeffb-sfdc) [reply page-objects/src/components/bottomBar/ProblemsView.ts:79]: @mlorinc do you mean like so?
```
    /**
     * Gets the count badge
     * @returns Promise resolving to the WebElement representing the count badge
     */
    async getCountBadgeAsync(): Promise<WebElement> {
        return await this.findElement(ProblemsView.locators.ProblemsView.changeCount);
    }

    /**
     * Gets the count badge
     * @returns Promise resolving to the WebElement representing the count badge
     */
     getCountBadge(): Promise<WebElement> {
        return this.findElement(ProblemsView.locators.ProblemsView.changeCount);
    }
```
I could make this change, but every other method within ProblemsView is in the form of
```
    async methodName: Promise<WebElement> {
        ...
    }
```

Comment (djelinek) [reply page-objects/src/components/bottomBar/ProblemsView.ts:79]: @mlorinc could you please answer here? thank you

**Review by djelinek — APPROVED** [review]

**Comment by jeffb-sfdc** [comment]
@odockal I think this PR is ready to be merged.  I don't have merge rights, so I'll need someone else to merge it.

**Comment by djelinek** [comment]
@jeffb-sfdc could you please do rebase on master branch?

I will wait till @mlorinc will do last code review and I will merge

**Review by mlorinc — APPROVED** [review]
I apologize for late answer. I approve all changes.

**Comment by djelinek** [comment]
@jeffb-sfdc could you please do a little bit of cleanup in PR commits? there are 5 commits and it would be nice to squash them together to keep healthy commit history

**Pull request merged**
